source/sphinx_extensions/plugin_directory/extension.py

```diff
@@ -84,8 +86,12 @@ def generate_rst(app):
                 template = env.get_template('action.rst')
                 rendered = template.render(title=title, cli_help=cli_help,
                                            api_help=api_help, bib_id=bib_id,
-                                           has_citations=bool(action.citations)
-                                           )
+                                           has_citations=bool(
+                                            action.citations),
+                                           plugin_cli_name=plugin_cli_name,
+                                           action_cli_name=action_cli_name,
```

Comment:
I don't think these two variables are used anymore: `plugin_cli_name` and `action_cli_name`.

Reply:
Sorry I missed that. Corrected. Thanks!